src/Symfony/Component/Messenger/Bridge/Redis/Transport/Connection.php

```diff
@@ -308,19 +308,15 @@ private function claimOldPendingMessages()
         $claimableIds = [];
         foreach ($pendingMessages as $pendingMessage) {
```

Comment:
on L303, we call
`$this->connection->xpending($this->stream, $this->group, '-', '+', 1);`

Doesn't the `1` means we ask for only one item? Of not, how much iterations can this loop have? If the number is possibly high, do we need to do anything about that?

Sorry if this question is dumb, I'm not into this topic much.